lms/templates/courseware/progress.html

```diff
@@ -80,7 +80,10 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
-                  %elif not is_id_verified or is_unverified:
+                  %elif missing_required_verification or is_unverified:
+                  ## The is_unverified variable comes from certificate status.
+                  ## At the moment, the assumption is we should not show generate certificate button
+                  ## if the certificate previously generated is in unverified status
                   <div class="msg-content">
                     ## Translators: This message appears to users when the users have not completed identity verification.
                     <h2 class="title">${_("Certificate unavailable")}</h2>
```

Comment:
@rlucioni The copy was created by @srpearce as part of ECOM-4007. Your suggested message sound more specifically associated with `unverified` generate cert case rather than a message that covers both the `missing_required_verification` and `unverified cert` case. Frankly, it's outside the scope of this bug fix as well.